onnxruntime/contrib_ops/cuda/bert/group_query_attention.cc

```diff
@@ -155,7 +155,6 @@ Status GroupQueryAttention<T>::ComputeInternal(OpKernelContext* context) const {
   bool use_memory_efficient_attention =
       !use_flash_attention &&
       !disable_memory_efficient_attention_ &&
-      local_window_size_ == -1 &&
       (sizeof(T) == 2 || parameters.sequence_length >= this->kernel_options_->MinSeqLenForEfficientAttentionFp32()) &&
```

Comment:
Also change line 161 - 164 since local attention can run in sm < 80 now.

BTW, I think we can remove the line of `(sizeof(T) == 2 || parameters.sequence_length >= this->kernel_options_->MinSeqLenForEfficientAttentionFp32()) &&` since there is no unfused kernel for GQA?